test/integration/clientset_test.go

```diff
@@ -219,6 +218,14 @@ func testBrokerClient(client servicecatalogclient.Interface, name string) error
 		)
 	}
 
+	// TODO: Here be dragons. Tests fail beyond this point due to known issues
+	// with our TPR-based storage implementation. Bail early (until those issues)
+	// are fixed, because some tests are better than no tests. If storage isn't
+	// TPR-based, carry on.
```

Comment:
this seems like a reason to pass the testing.T in so we can explicitly call `t.Skip("reason")` along with all of the other error and logging functions.